api/database/seeders/DatabaseSeeder.php

```diff
@@ -64,4 +64,43 @@ private function truncateTables()
         PoolCandidateSearchRequest::truncate();
         User::truncate();
     }
+
+    // set up some users to allow authentication
+    private function loadUsers()
+    {
+        // shared Laravel auth user
+        User::factory()->create([
+            'email' => 'admin@test.com',
+            'sub' => 'admin@test.com',
+            'roles' => ['ADMIN']
+        ]);
+
+        User::factory()->create([
+            'sub' => '4810df0d-fcb6-4353-af93-b25c0a5a9c3e', // SiC to localhost - PG
+            'roles' => ['ADMIN']
```

Comment:
I think it would be useful to set emails as well, or at least names, to be able to identify one's own user when looking through the database.

Reply:
Yeah, I see your point about needed to identify a row.  At the same time I wanted to be a little discreet about it since it's a public repo.  I figured that our github usernames are already public so I built fake email addresses with those and pushed that.  Will that solve the difficulty?

Reply:
Yeah these fake emails based on gh username are perfect 👌

Reply:
Not sure if helpful, but can be auto-generated from public org member list here: https://api.github.com/orgs/GCTC-NTGC/members

Another API endpoint should do it from a team.